doc/htmldoc/index.rst

```diff
@@ -75,6 +83,36 @@ Welcome to the NEST Simulator documentation
 
              Discover all our examples!
 
+.. grid::
+
+    .. grid-item::
+        :columns: 6
+        :child-direction: row
+        :child-align: end
+
+        .. raw:: html
+
+           <object width="89%" height="89%" data="_static/img/excalidraw_nestconcept.svg" type="image/svg+xml"></object>
```

Comment:
I wonder if it would make sense to flip the figure by 90 degrees, with "Protocols" -> ... -> "Analysis" on top left-to-right and the other three boxes below. At least on a computer screen that seems to be more effective use of screen real estate.